Formula/trailscraper.rb

```diff
@@ -21,31 +20,31 @@ class Trailscraper < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "bd9a48aeb5cd08a08fb10d35d83eebef69c66e9436de70e737e3c10e3efbc3f8"
   end
 
-  depends_on "python@3.11"
```

Comment:
why not `python@3.11`?

Reply:
And this is supported in the CI, right?
https://github.com/flosell/trailscraper/commit/510c9abf25c4e81385f4b46618b731417d4c75e4